buyer.go

```diff
@@ -146,18 +152,48 @@ func newTicketPurchaser(cfg *config,
 		priceMode = AvgPriceDualMode
 	}
 
+	// Determine if there have been existing runs and use buy period
+	f, err := os.Open(filepath.Join(csvPath, csvPurchasedFn))
+	if err != nil {
+		log.Errorf("Error opening file: %v", err)
```

Comment:
Does this cause the program to crash if the file doesn't exist? Check for it existing first.